frontend/src/v5/ui/v4Adapter/resuableOverrides.styles.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { css } from 'styled-components';
+
+export const primaryButtonStyling = css`
```

Comment:
Just a thought. Do you think it's worth using these in the V4 overrides where we've 'hacked' in a button? e.g. userList.overrides FloatingButton. 

I'm thinking it's probably not worth it as these are temporary patches before the realV5 components are added, and ultimately it only clears up a few lines